messages/commonFlags.md

```diff
@@ -72,6 +72,28 @@ Number of minutes to wait for command to complete and display results.
 
 Username or alias of the target org.
 
+# flags.job-id.summary
+
+Job ID of the deploy operation you want to resume.
+
+# flags.job-id.description
+
+These commands return a job ID if they time out or you specified the --async flag:
+
+- sf deploy pipeline
```

Comment:
Maybe use _<%= config.bin %> <%= command.id %>_ and we can reuse it for promotes in a future ?

Reply:
In this case I don't think is necessary because here we want to list all commands that can be resumed. SO in the future when we have another commands we can add the to the list, not to repleace it.